src/test/java/com/theironyard/AuthenticationControllerTests.java

```diff
@@ -0,0 +1,167 @@
+package com.theironyard;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.theironyard.commands.LoginCommand;
+import com.theironyard.entities.User;
+import com.theironyard.repositories.UserRepository;
+import com.theironyard.utilities.PasswordStorage;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class AuthenticationControllerTests {
+    User user1;
+    User user2;
+    String username;
+    String password;
+
+    @Autowired
+    UserRepository userRepo;
+
+    @Autowired
+    WebApplicationContext wap;
+
+    MockMvc mockMvc;
+
+    @Before
+    public void before() throws PasswordStorage.CannotPerformOperationException {
+        mockMvc = MockMvcBuilders.webAppContextSetup(wap).build();
+
+        username = "TestUsername";
```

Comment:
Given that you are only setting the user to a class member variable I would say just do your `new User` with all data in the constructor.